doc/workshop/optimizer/GeneticAlgorithms/Inputs/tests

```diff
@@ -2,6 +2,7 @@
   [./GA_MaxwRep]
     type = 'RavenFramework'
     input = '1_GA_MaxwRep.xml'
+    image = 'MaxwReplacement/plotter1.png'
```

Comment:
I guess you need to provide a tolerance for image diff, since this test failed on windows. @mandd

Reply:
Fixed